unit_testing/test_register.py

```diff
@@ -167,11 +189,32 @@ def test_register_step_ants_registration(step2_data):
      verbose=cli_params.verbose,
     )
 
-@pytest.mark.skip(reason="TODO")
-def test_register_step_ants_slice_regularized_registration():
-     """
-     """
-     raise NotImplementedError()
+
```

Comment:
There should be at least two other tests to cover the two other algorithms 'None' and 'Random'. They obviously can't test for exact results, but it would be good to check they at least run.

Reply:
agreed-- could you pls open an issue? i have no more time to dedicate to this PR